docs/PublishingGuide.md

```diff
@@ -53,6 +73,14 @@ Alternatively, you can manually trigger an update.
 1. Go to [your fork in Travis-CI](https://travis-ci.org/search/publish-RepoSense/), click on **More options** on the right then **Trigger build**
 1. In the pop up, click **Trigger custom build**
 
+#### Using GitHub Actions
+
+1. Edit [main.yml](../../../../publish-RepoSense/edit/master/.github/workflows/main.yml) and uncomment the `schedule:` section
+1. You may change the expression after `cron:` to a schedule of your choice; read more about cron syntax [here](https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+1. Commit your changes
+
+Alternatively, you can trigger an update by pushing an empty commit to your fork. GitHub Actions currently do not support manual triggers.
```

Comment:
Maybe change to `Currently, the GitHub Actions UI doesn't support the manual execution of workflows`